packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -23,175 +23,182 @@ import * as integrationHelpers from '../util/helpers';
 
 const apiDescribe = integrationHelpers.apiDescribe;
 
-apiDescribe.only('Database transactions', (persistence: boolean) => {
-  type TransactionStep = (
+apiDescribe('Database transactions', (persistence: boolean) => {
+  type TransactionStage = (
     transaction: firestore.Transaction,
     docRef: firestore.DocumentReference
   ) => void;
 
   /**
-   * Used for testing that all possible combinations of executing transactions
-   * result in the desired document value or error.
-   *
    * The transaction steps are postfixed by numbers to indicate the calling
```

Comment:
s/steps/stages/. Also maybe "These transaction stages" or "The transaction stages that follow" or something like that.